Web-Scraping/amazon_reviews_scraping/README.md

```diff
@@ -0,0 +1,24 @@
+## USE: As a part of projects where you are required to do sentiment analysis on customer reviews data. </br>
+
+Scrapy is a web crawling framework for a developer to write code to create, which defines how a particular site (or a group of websites) will be scrapped.
+
+## Steps:</br>
```

Comment:
Use three sharps('#') in Steps section!

Rest LGTM :+1:

Reply:
Done.

Reply:
Should I close this pull request and open another from a branch?